### ISSUE TITLE
Slider | Material - Thumb label is not appearing 

### ISSUE THREAD

**Issue by thanecoder** [ngx-formly/ngx-formly#3560]
**Description**
Facing one issue in the new version - 6.1.0, the Slider thumbLabel is not coming. 
I have set the thumbLabel as true in the props , but still the thumbLabel is not appearing.

**Minimal Reproduction**
Stackblitz link for this issue - https://stackblitz.com/edit/angular-brfkcx?file=src%2Fapp%2Fapp.component.ts.

**Your Environment**

- Angular version: 15
- Formly version: 6.1.0

**Comment by digoburigo** [comment]
I don't if this error is related, but when I start my app when loading the ngx-formly I'm getting this error
```
ngx-formly-core.mjs:417 An error occurred while destroying the Formly component type "slider" TypeError: Cannot read properties of undefined (reading 'removeEventListener')
    at MatSliderVisualThumb.ngOnDestroy (slider.mjs:148:15)
    at executeOnDestroys (core.mjs:6031:32)
    at cleanUpView (core.mjs:5941:9)
    at destroyViewTree (core.mjs:5767:21)
    at destroyLView (core.mjs:5919:9)
    at RootViewRef.destroy (core.mjs:11899:9)
    at ComponentRef.destroy (core.mjs:12320:23)
    at FormlyConfig.resolveFieldTypeRef (ngx-formly-core.mjs:414:26)
    at componentRefInstance (ngx-formly-core.mjs:2128:44)
    at Object.get (ngx-formly-core.mjs:2138:35)
resolveFieldTypeRef @ ngx-formly-core.mjs:417
```
I'm going to see if I can reproduce this error in a new project

**Comment by thanecoder** [comment]
Hi @aitboudad, Kindly suggest a workaround for both the above-mentioned issues.

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v6.1.1](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.1.1) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!

**Note**:  for the following error is related to material slider issue, will send a fix when time allows :pray::
```
ngx-formly-core.mjs:417 An error occurred while destroying the Formly component type "slider" TypeError: Cannot read properties of undefined (reading 'removeEventListener')
```

**Comment by alexdalgleishmorel** [comment]
@aitboudad this issue is still present for me with Formly 6.2.2 and Angular 16.

**Comment by alexdalgleishmorel** [comment]
@aitboudad this issue is still persisting with 'ngx-formly/core@6.3.0' and Angular 16.

**Comment by Tze-Sien** [comment]
This error still exist in  "@ngx-formly/material": "^6.1.8".

It only happened on the initial load of the component, based on what I observed, the ngOnDestroy will be called on the initial load of the component ( A bit weird, looks like it will be created -> destroy -> create ) 

`An error occurred while destroying the Formly component type "slider" TypeError: Cannot read properties of undefined (reading 'removeEventListener')
    at _MatSliderVisualThumb.ngOnDestroy (slider.mjs:195:11)
    at executeOnDestroys (core.mjs:8416:32)
    at cleanUpView (core.mjs:8324:9)
    at destroyViewTree (core.mjs:8156:21)
    at destroyLView (core.mjs:8301:9)
    at ViewRef$1.destroy (core.mjs:13877:9)
    at ComponentRef.destroy (core.mjs:15662:23)
    at FormlyConfig.resolveFieldTypeRef (ngx-formly-core.mjs:465:20)
    at componentRefInstance (ngx-formly-core.mjs:2677:38)
    at Object.get (ngx-formly-core.mjs:2686:27)`

**Comment by Tze-Sien** [comment]
> This issue has been fixed and released as part of [v6.1.1](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.1.1) release.
> 
> Please let us know, in case you are still encountering a similar issue/problem. Thank you!
> 
> **Note**: for the following error is related to material slider issue, will send a fix when time allows 🙏:
> 
> ```
> ngx-formly-core.mjs:417 An error occurred while destroying the Formly component type "slider" TypeError: Cannot read properties of undefined (reading 'removeEventListener')
> ```



This error still exist in  "@ngx-formly/material": "^6.1.8".

It only happened on the initial load of the component, based on what I observed, the ngOnDestroy will be called on the initial load of the component ( A bit weird, looks like it will be created -> destroy -> create ) 

Is this issue related to the mechanism of how formly render the component from the formlyFieldConfigs? I can help with that if possible. @aitboudad 

`An error occurred while destroying the Formly component type "slider" TypeError: Cannot read properties of undefined (reading 'removeEventListener')
    at _MatSliderVisualThumb.ngOnDestroy (slider.mjs:195:11)
    at executeOnDestroys (core.mjs:8416:32)
    at cleanUpView (core.mjs:8324:9)
    at destroyViewTree (core.mjs:8156:21)
    at destroyLView (core.mjs:8301:9)
    at ViewRef$1.destroy (core.mjs:13877:9)
    at ComponentRef.destroy (core.mjs:15662:23)
    at FormlyConfig.resolveFieldTypeRef (ngx-formly-core.mjs:465:20)
    at componentRefInstance (ngx-formly-core.mjs:2677:38)
    at Object.get (ngx-formly-core.mjs:2686:27)`
